### ISSUE TITLE
Azimuthal Integral Tutorial - Case 2 not working

### ISSUE THREAD

**Issue by andy-bridger** [pyxem/pyxem-demos#58]
npt_rad no longer keyword
other issues associated with dp.get_azimuthal_integral1d(npt_rad =100, detector_dist=camera_length, detector=detector)

**Comment by pc494** [comment]
It looks like `npt_rad` has been replaced by `npt` in the function signature, which should resolve the first problem.

**Comment by andy-bridger** [comment]
Yeh, tracked that down, the rest of the issues seemed to be with the specifying a detector and detector distance within this method. I think there has been a change in the code and now there is a set_ai method, I'm not certain exactly how it has changed but it no longer functions as shown in the demo

**Comment by pc494** [comment]
Okay, sounds like I will need to rewrite it then, I hadn't noticed the scale of those changes; thanks for the bug report.

**Comment by pc494** [comment]
@CSSFrancis I had a look at this today (both demo 07 and 09) - is there any chance these are easy fixes for you?

**Comment by CSSFrancis** [comment]
Yes. I can make the changes to the demo (hopefully today but definitely tomorrow). I think that this demo never got updated with the latest changes (my fault).
